src/pyOpenMS/pyopenms/dataframes.py

```diff
@@ -293,20 +293,26 @@ def get_key(val):
     cv.loadFromOBO("psims", File.getOpenMSDataPath() + "/CV/psi-ms.obo")
     clearMVs = [cv.getTerm(m).name if m.startswith("MS:") else m for m in decodedMVs]
     #cols = ["id", "RT", "mz", "score", "charge"] + decodedMVs
-    clearcols = ["id", "RT", "mz", mainscorename, "charge"] + clearMVs
-    coltypes = ['U100', 'f', 'f', 'f', 'i'] + types
+    clearcols = ["id", "RT", "mz", mainscorename, "charge", "protein_accession", "start", "end"] + clearMVs
+    coltypes = ['U100', 'f', 'f', 'f', 'i','U100', 'U100', 'U100'] + types
```

Comment:
I could imagine that these strings can be longer than 100 characters

Reply:
Would getting the maximum string length in each pep evidence column and setting the dtype accordingly work?

Reply:
yes that would work.
I just wonder if U100 saves memory in cases that we have e.g., only U20 strings.
e.g., would str be more memory efficient?